pkgs/by-name/li/libbytesize/package.nix

```
@@ -51,6 +54,7 @@ stdenv.mkDerivation (finalAttrs: {
     pcre2
   ];
 
+  doInstallCheck = stdenv.buildPlatform.canExecute stdenv.hostPlatform;
```

```suggestion
  doInstallCheck = true;
```

https://github.com/NixOS/nixpkgs/blob/fe6a9751e6a83b2e4297eea17b89cb294eeb8cd6/pkgs/stdenv/generic/make-derivation.nix#L249-L250